template/content/clients.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
```

Comment:
Actually; I want the logger to be a service over hyperbahn soon (tm).

I'll stick to `clients` for now in the term of "clients to other services and pieces of infrastructure"